tensorflow/lite/micro/memory_planner/greedy_memory_planner.cc

```diff
@@ -356,28 +377,21 @@ void GreedyMemoryPlanner::PrintMemoryPlan(ErrorReporter* error_reporter) {
         continue;
       }
       const int size = requirements->size;
+      memory_use += size;
       const int line_start = (offset * kLineWidth) / max_size;
       const int line_end = ((offset + size) * kLineWidth) / max_size;
       for (int n = line_start; n < line_end; ++n) {
         if (line[n] == '.') {
-          char display;
-          if (i < 10) {
-            display = '0' + i;
-          } else if (i < 36) {
-            display = 'a' + (i - 10);
-          } else if (i < 62) {
-            display = 'A' + (i - 36);
-          } else {
-            display = '*';
-          }
-          line[n] = display;
+          line[n] = GetOrdinalCharacter(i);
         } else {
           line[n] = '!';
         }
       }
     }
     line[kLineWidth] = 0;
-    TF_LITE_REPORT_ERROR(error_reporter, "%s", (const char*)line);
+
+    TF_LITE_REPORT_ERROR(error_reporter, "%s%d: %s (%dk)", t < 10 ? " " : "", t,
+                         (const char*)line, (memory_use + 1023) / 1024);
```

Comment:
use MicroPrintf instead.

and I believe that the error_reporter param can be dropped from PrintMemoryPlan with this change.